include/flx/forward_automatic/newton_raphson.hpp

```diff
@@ -0,0 +1,61 @@
+//==================================================================================================
+/*
+  EVE - Expressive Vector Engine
+  Copyright : EVE Contributors & Maintainers
+  SPDX-License-Identifier: MIT
+*/
+//==================================================================================================
+#pragma once
+
+#include <eve/detail/overload.hpp>
+#include <flx/forward_automatic/var.hpp>
+#include <flx/forward_automatic/valder.hpp>
+#include <flx/forward_automatic/val.hpp>
+#include <flx/forward_automatic/der.hpp>
+
+namespace eve
+{
+  EVE_REGISTER_CALLABLE(newton_raphson_);
+  EVE_DECLARE_CALLABLE(newton_raphson_,newton_raphson);
+  EVE_CALLABLE_API(newton_raphson_,newton_raphson);
+}
+
+
+namespace flx
+{
+  inline eve::detail::callable_object<eve::tag::newton_raphson_> const newton_raphson = {};
```

Comment:
Missing doc

Reply:
in fact it was work in progress and I wanted to put that in a tutorial. in remove the related files temporarily and will see later. The existing version is to poor to be in the library.